roles/reproducer/tasks/main.yml

```diff
@@ -104,6 +104,15 @@
         - bootstrap_env
         - bootstrap_layout
 
+- name: Bootstrap nat64 if needed
+  when:
+    - cifmw_use_libvirt | default(false) | bool
+    - cifmw_use_nat64 | default(false) | bool
+  tags:
+    - bootstrap_nat64
```

Comment:
that should probably be part of `bootstrap_layout` and `bootstrap_env`

Reply:
Done